wsmaster/che-core-sql-schema/src/main/resources/che-schema/7.15.0/1__add_devfile_component_automount_workspace_secrets.sql

```diff
@@ -0,0 +1,13 @@
+--
```

Comment:
7.16?

Reply:
As far as I understand the weekly build versioning, it will be Z release every inside-sprint week and Y release every sprint. So `7.15.0` -> `7.15.1` -> `7.15.2` -> `7.16.0`. This will make this into `7.15.1`